File-Formats/PDF/Working-with-Annotations.md

```diff
@@ -12269,4 +12269,9 @@ freeText.SetAppearance(true);
 </td>
 </tr>
 </table>
-</table>
\ No newline at end of file
+</table>
+
+## See Also
+[How to modify the size of C# PDF Annotation](https://www.syncfusion.com/kb/10290/how-to-modify-the-size-of-the-annotation-in-pdf-using-c-and-vb-net)
+
+N> You can also explore our [C# PDF Annotation](https://www.syncfusion.com/document-processing/pdf-framework/net/pdf-library/pdf-annotation) feature tour page that how to use .NET PDF with 20+ annotation types with built-in support, or create their own custom annotations and [PDF Annotaions .NET demo](https://ej2.syncfusion.com/aspnetmvc/PDF/AnnotationFlatten#/material) that demonstrates the creation of different types of annotations such as ink, free text, pop up, text markup annotation, and more.
```

Comment:
Get the content review for this
![image](https://user-images.githubusercontent.com/61148953/186588565-264bb020-741a-4130-b0ca-25adb5588c6d.png)

Reply:
got it